### PR TITLE
convert to onEvent

### PR DESCRIPTION
**Pull request by tiina303**
in place of https://github.com/PostHog/posthog-braze-plugin/pull/12 on top of https://github.com/PostHog/posthog-braze-app/pull/1

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -188,74 +186,26 @@ const _generateBrazeRequestBody = (pluginEvent: PluginEvent, meta: BrazeMeta): B
...
 
-    // NOTE: Send all requests in parallel, error responses already handled and logged by fetchBraze
-    await Promise.all(brazeRequests)
-
     const elapsedTime = (Date.now() - startTime) / 1000
-
-    if (elapsedTime >= 30) {
-        console.warn(`🐢🐢 Slow exportEvents warning. Export took ${elapsedTime} seconds.`)
-    }
+    console.log(`🚀 Exported 1 event to Braze in ${elapsedTime} seconds.`)
```
Comment (paolodamico) [inline index.ts:210]: potential log spam?

File: tests/export-events.test.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// This App supports pushing events to Braze also, via the `exportEvents` hook. It
+// This App supports pushing events to Braze also, via the `onEvent` hook. It
```
Comment (paolodamico) [inline tests/export-events.test.ts:1]: nit: rename file name

File: index.ts (hunk 1 above)
Comment (tiina303) [reply index.ts:210]: we have no other way to know how many events we actually sent atm

File: tests/export-events.test.ts (hunk 2 above)
Comment (tiina303) [reply tests/export-events.test.ts:1]: it's a fine name, we're still exporting events 😅

**Pull request merged**